### DISCUSSION TITLE
Datasource not found?

### DISCUSSION THREAD

**Discussion by gm5dna** [magicbug/Cloudlog#1626] [Q&A]
Hi there,

I'm a complete novice, but have been running Cloudlog successfully in an Ubuntu VM for a few months now.

I'm now getting this error on booting up though:

![Screenshot 2022-09-30 at 21 28 59](https://user-images.githubusercontent.com/18617879/193351626-dc3af009-7ced-4c91-8e02-a6affe09d13c.png)

Any help would be much appreciated!

Thanks,

Stuart

**Comment by phl0** [comment]
That's got nothing to do with Cloudlog. That is a topic of the operating system.

**Reply by gm5dna** [reply]
Thanks

**Comment by gm5dna** [comment]
Well everything seem to have sorted itself now... not sure what was going on there... :-/
